insights/client/connection.py

```diff
@@ -170,13 +170,28 @@ def _http_request(self, url, method, log_response_text=True, **kwargs):
         Returns
             HTTP response object
         '''
-        logger.log(NETWORK, "%s %s", method, url)
+        log_message = "{method} {url}".format(method=method, url=url)
+        if "data" in kwargs.keys():
+            log_message += " data={data}".format(data=kwargs["data"])
+        if "json" in kwargs.keys():
+            log_message += " json={json}".format(json=json.dumps(kwargs["json"]))
+        if "headers" in kwargs.keys():
+            log_message += " headers={headers}".format(headers=kwargs["headers"])
+        if "files" in kwargs.keys():
+            attachments = []
+            for name, content in six.iteritems(kwargs["files"]):
+                if isinstance(content, tuple):
+                    attachments.append("{name}:{file}".format(name=name, file=content[0]))
+                else:
+                    attachments.append(name)
+            log_message += " attachments={files}".format(files=",".join(attachments))
+        logger.log(NETWORK, log_message)
         try:
             res = self.session.request(url=url, method=method, timeout=self.config.http_timeout, **kwargs)
         except Exception:
             raise
         logger.log(NETWORK, "HTTP Status: %d %s", res.status_code, res.reason)
-        if log_response_text or res.status_code != 200:
+        if log_response_text or res.status_code // 100 != 2:
```

Comment:
hm I think it is still useful to log the response text in 2xx responses different than 200

Reply:
The `log_response_text` is `True` by default, so there's no other way to silence output of API endpoints that return `201 Created` (checkin, for example).
There's nothing interesting in the checkin response, it's just a big JSON blob of host data, including the verbose system profile.